<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingWindow</class>
 <widget class="QMainWindow" name="TrainingWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources.qrc">
    <normaloff>:/images/Logo.png</normaloff>:/images/Logo.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">#centralwidget, QDialog
{
    background-image: url(:/images/Background1.png);
}

QPushButton
{
    color:rgb(200,200,200,200);
    background-color:rgb(180,180,180,100);
    border-radius:3;
}

QPushButton:checked
{
    color:rgb(200,200,200,200);
    background-color:rgb(140,140,200,100);
    border-radius:3;
}

QLabel
{
    color:rgb(200,200,200,200);
    background-color:rgb(150,150,150, 100);
    border-radius:3;
}

#scrollPane, #paymentsScrollPane
{
    color:rgb(100,100,100,200);
    background-color:rgb(150,150,150,100);
    border-radius:3;
}

QScrollBar:horizontal, QScrollBar:vertical
{
    height:50px;
    width:50px;
    background-color:rgb(120, 120, 120, 140);
}

QScrollBar::add-page:horizontal, QScrollBar::add-page:vertical, QScrollBar::sub-page:horizontal, QScrollBar::sub-page:vertical
{
    background:none;
}


QComboBox QAbstractItemView
{
    background-color: rgb(110, 110, 110);
    selection-background-color: rgb(80, 80, 80);
}

QComboBox
{
    min-height:30px;
    color: rgb(200, 200, 200, 200);
    background-color: rgb(110, 110, 110, 150);
    selection-background-color: rgb(80, 80, 80, 150);
}

QMessageBox
{
    background-color: rgb(180, 180, 180);
    background-image: url();
}
QMessageBox QLabel
{
    color:rgb(0,0,0);
    background-color:rgb(0,0,0,0);
}
QMessageBox QPushButton
{
    color:rgb(0,0,0);
    background-color:rgb(140,140,140);
    padding: 20px;
}
</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>250</width>
      <height>210</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="cameraDisplay">
    <property name="geometry">
     <rect>
      <x>15</x>
      <y>15</y>
      <width>240</width>
      <height>180</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="nameLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>520</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>User: &lt;name&gt;</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>360</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="mainButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>150</y>
      <width>191</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Start Search</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="testButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>240</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>240</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="statusLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>80</y>
      <width>360</width>
      <height>80</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>Please press 'Start Search' to begin</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="scrollPane">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>300</y>
      <width>780</width>
      <height>120</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::NoFrame</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>0</y>
       <width>110</width>
       <height>110</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(100,100,100,200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>5</y>
       <width>100</width>
       <height>100</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(0,0,0);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QScrollBar" name="horizontalScrollBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>780</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="ledsButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>180</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>IR LEDs</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>230</y>
      <width>310</width>
      <height>70</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="exitButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>240</y>
      <width>130</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="defaultButton">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Set as default</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="deleteButton">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>240</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 95, 95, 240);</string>
    </property>
    <property name="text">
     <string>Delete</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="autoSnapButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>90</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AutoSnap</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="autoAddButton">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>135</y>
      <width>130</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>AutoAdd</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>80</y>
      <width>150</width>
      <height>140</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="faceCountLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>193</y>
      <width>250</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>No faces found</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="faceDisplay">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>100</y>
      <width>100</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="addButton">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>160</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLabel" name="countdownLabel">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>90</y>
      <width>230</width>
      <height>70</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>32</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(0,0,0,0);</string>
    </property>
    <property name="text">
     <string>3</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>230</y>
      <width>160</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>230</y>
      <width>290</width>
      <height>60</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <zorder>label_13</zorder>
   <zorder>label_12</zorder>
   <zorder>label_8</zorder>
   <zorder>label_11</zorder>
   <zorder>label_3</zorder>
   <zorder>cameraDisplay</zorder>
   <zorder>nameLabel</zorder>
   <zorder>label_7</zorder>
   <zorder>mainButton</zorder>
   <zorder>saveButton</zorder>
   <zorder>statusLabel</zorder>
   <zorder>scrollPane</zorder>
   <zorder>horizontalScrollBar</zorder>
   <zorder>ledsButton</zorder>
   <zorder>exitButton</zorder>
   <zorder>defaultButton</zorder>
   <zorder>deleteButton</zorder>
   <zorder>autoSnapButton</zorder>
   <zorder>autoAddButton</zorder>
   <zorder>faceCountLabel</zorder>
   <zorder>faceDisplay</zorder>
   <zorder>addButton</zorder>
   <zorder>countdownLabel</zorder>
   <zorder>testButton</zorder>
  </widget>
 </widget>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections/>
</ui>
